adev/angular.json

```
@@ -41,7 +41,10 @@
               "src/robots.txt",
               "src/assets",
               "src/llms.txt",
-              "src/llms-full.txt"
+              "src/llms-full.txt",
+              "src/context/airules.md",
+              "src/context/angular-20.mdc",
+              "src/context/best-practicesd.md"
```

nit: we can have the whole directory instead of the 3 separate entries. 
```suggestion
              "src/context/"
```